globus_sdk/authorizers/refresh_token.py

```diff
@@ -1,19 +1,12 @@
 import warnings
 import time
+
 from globus_sdk.authorizers.base import GlobusAuthorizer
 
-# a percent error expected in the `expires_in` values on AccessTokens,
-# expressed as a fractional value
-# taken out of the expiration value, not the total time (because that would be
-# weird, yeah?)
-# This is 5%, not 0.05%!
-_EXPIRATION_PERCENT_ERROR = 0.05
-# this is a fixed number of seconds to be pulled down from the expiration
-# timer, so that when the expires_in value is very small, the percent error
-# doesn't go to 0 and make us run the timer all the way down
-# so, we want (NOW - expires_in - (expires_in*PCT_ERROR) - FIXED_ERROR)
-# ultimately
-_EXPIRATION_FIXED_ERROR = 5
+
+# Provides a buffer for token expiration time to account for
+# possible delays or clock skew.
+EXPIRES_ADJUST_SECONDS = 60
```

Comment:
I think we spoke about this, but I just wanted to confirm that I'm cool with this change.
If it ends up not being enough, we can deal with it then.